core/src/main/scala/org/scalafmt/config/Spaces.scala

```diff
@@ -6,5 +6,6 @@ import metaconfig.ConfigReader
 case class Spaces(
     beforeContextBoundColon: Boolean = false,
     afterTripleEquals: Boolean = false,
-    inImportCurlyBraces: Boolean = false
+    inImportCurlyBraces: Boolean = false,
+    neverAroundInfixTypes: Seq[String] = Seq("##")
```

Comment:
Agree! I was expecting this ^_^ but was not sure what is better as default value.